rest/taskana-rest-spring/src/main/java/pro/taskana/workbasket/rest/WorkbasketController.java

```diff
@@ -214,76 +262,111 @@
-      getWorkbasketAccessItems(@PathVariable(value = "workbasketId") String workbasketId)
-          throws NotAuthorizedException, WorkbasketNotFoundException {
+  public ResponseEntity<WorkbasketAccessItemCollectionRepresentationModel> getWorkbasketAccessItems(
+      @PathVariable(value = "workbasketId") String workbasketId)
+      throws NotAuthorizedException, WorkbasketNotFoundException {
     LOGGER.debug("Entry to getWorkbasketAccessItems(workbasketId= {})", workbasketId);
 
     List<WorkbasketAccessItem> accessItems =
         workbasketService.getWorkbasketAccessItems(workbasketId);
-    final ResponseEntity<TaskanaPagedModel<WorkbasketAccessItemRepresentationModel>> result =
+    final ResponseEntity<WorkbasketAccessItemCollectionRepresentationModel> result =
         ResponseEntity.ok(
-            workbasketAccessItemRepresentationModelAssembler.toPageModelForSingleWorkbasket(
-                workbasketId, accessItems, null));
+            workbasketAccessItemRepresentationModelAssembler
+                .toTaskanaCollectionModelForSingleWorkbasket(workbasketId, accessItems));
     if (LOGGER.isDebugEnabled()) {
       LOGGER.debug("Exit from getWorkbasketAccessItems(), returning {}", result);
     }
 
     return result;
   }
 
+  /**
+   * This endpoint replaces all Workbasket Access Items for a given Workbasket with the provided
+   * ones.
+   *
+   * @title Set all Workbasket Access Items
+   * @param workbasketId the id of the Workbasket whose Workbasket Access Items will be replaced
+   * @param workbasketAccessResourceItems the new Workbasket Access Items.
+   * @return the new Workbasket Access Items for the requested Workbasket
+   * @throws NotAuthorizedException if the current user is not member of role BUSINESS_ADMIN or
+   *     ADMIN
+   * @throws InvalidArgumentException if the new Workbasket Access Items are not provided.
+   * @throws WorkbasketNotFoundException TODO: this is never thrown.
+   * @throws WorkbasketAccessItemAlreadyExistException if a duplicate Workbasket Access Item
+   *     exists in the provided list.
+   */
   @PutMapping(path = RestEndpoints.URL_WORKBASKET_ID_ACCESS_ITEMS)
   @Transactional(rollbackFor = Exception.class)
-  public ResponseEntity<TaskanaPagedModel<WorkbasketAccessItemRepresentationModel>>
-      setWorkbasketAccessItems(
-          @PathVariable(value = "workbasketId") String workbasketId,
-          @RequestBody List<WorkbasketAccessItemRepresentationModel> workbasketAccessResourceItems)
-          throws NotAuthorizedException, InvalidArgumentException, WorkbasketNotFoundException,
-              WorkbasketAccessItemAlreadyExistException {
+  public ResponseEntity<WorkbasketAccessItemCollectionRepresentationModel> setWorkbasketAccessItems(
+      @PathVariable(value = "workbasketId") String workbasketId,
+      @RequestBody WorkbasketAccessItemCollectionRepresentationModel workbasketAccessResourceItems)
```

Comment:
strange name...1. this is not resource but model now, right? and 2. the resource/model before items? if at all, this should be workbasketAccessItemResource instead of workbasketAccessResourceItems , but workbasketAccessItemCollectionRepresentationModel would be more accurate I think